UDV-Core/src/Utils/BaseDemo/js/BaseDemo.js

```diff
@@ -15,15 +15,10 @@ export class BaseDemo {
         this.authService;
         this.config = {};
         this.parentElement;
-        this.view;  // itowns view (3d scene)
-        this.extent;  // itowns extent (city limits)
+        this.view; // itowns view (3d scene)
+        this.extent; // itowns extent (city limits)
         this.controls;
-        /**
-         * Object used to manage the 3DTiles layer.
-         *
-         * @type {TilesManager}
-         */
-        this.tilesManager;
+        this.itowns = itowns;
```

Comment:
You don't need to add this.itowns as an attribute of baseDemo since you set it to a value that is already accessible here.